src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpExporter.cs

```diff
@@ -134,5 +128,11 @@ protected override bool OnShutdown(int timeoutMilliseconds)
 
             return Task.WaitAny(new Task[] { this.channel.ShutdownAsync(), Task.Delay(timeoutMilliseconds) }) == 0;
         }
+
+        /// <inheritdoc/>
+        protected override void OnTracerProviderSet(TracerProvider tracerProvider)
```

Comment:
This is based on the assumption that exporter is owned by one provider - which could be the case today (given provider Dispose will just clean up all the exporters) but we don't know if it would hold true in the future.

Reply:
Don't have an answer for this! If you used the same Exporter in multiple TracerProviders that would be a problem, yes. BUT, I will say this: It's more than an assumption, more like a rule. Currently Zipkin, Jaeger, & Otlp Exporters you cannot new up. You have to use the builder extension to add them to the provider you are building up. So I think in order to get yourself into the situation above you would have to work really hard (reflection, etc.) or use a third-party Exporter which doesn't follow that pattern.

Reply:
I agree.